### PR TITLE
Regression fix: shared libraries in autoconf projects on Linux host

### PR DESCRIPTION
**Pull request by brion**
Partial revert of commit 53a969dd, plus addition of source comment explaining why the verbiage about ld compatibility is there.

### CODE REVIEW

**Comment by kripken** [comment]
Thanks, and sorry for the regression.

Apparently we don't have a test for this. Can you please add one? Something simple in `tests/test_other.py` is enough. Even better would be some interesting usage of autotools, if that's feasible.

Please make the pull to the incoming branch, and add yourself to AUTHORS.

**Comment by brion** [comment]
Good suggestion to add tests, I actually messed up the patch! The 'GNU ld' bit needs to be on the -v not the --version output. Will fix shortly and add the tests.

**Comment by brion** [comment]
Ok, now with correct fix, AUTHORS update, and a regression test.

File: emcc [hunk 1]
```diff
@@ -172,7 +172,9 @@ There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR P
   exit(0)
 
 elif len(sys.argv) == 2 and sys.argv[1] == '-v': # -v with no inputs
-  print 'emcc (Emscripten gcc/clang-like replacement) %s' % shared.EMSCRIPTEN_VERSION
+  print '''emcc (Emscripten gcc/clang-like replacement + linker emulating GNU ld) %s
+(autoconf likes to see GNU above to enable shared object support)
```
Comment (kripken) [inline emcc]: please move the comment in parens into a python comment

**Comment by brion** [comment]
updated, moved comment to comment from output per comment :)

**Comment by kripken** [comment]
Merged, thanks!

**Pull request closed without merging**